src/puppeteer/puppeteer-plugin.ts

```diff
@@ -17,6 +17,8 @@ export class PuppeteerPlugin extends BrowserPlugin<typeof Puppeteer> {
             useIncognitoPages,
         } = launchContext;
 
+        const typedAnonymizedProxyUrl = anonymizedProxyUrl as string | undefined;
```

Comment:
i guess the `anonymizedProxyUrl` comes from the index signature on `LaunchContext`, right? would be great to redesign the typings so we see actual options and their types, I believe it should be possible (but not in this PR)